charts/line-chart/src/LineChart.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Chart, Line } from '@lg-charts/core';
+
+import { LineChartProps } from './LineChart.types';
+
+export function LineChart({ data }: LineChartProps) {
+  return (
+    <Chart>
+      {data.map(({ name, data }) => (
+        <Line name={name} data={data} key={name} />
```

Comment:
Names are currently not technically guaranteed to be unique. Could concat with the index, but seeing that index usage could also be problematic, just went with name since they probably _should_ be unique, given that they ID a given line on a chart. Curious if there's a specific pattern we typically follow

Reply:
If echarts expects to identify a series based on the `name` then I'd say that'd good enough for our React keys. Maybe add a comment here explaining that? 
I agree using `index` isn't great, because that could lead to bugs if lines get reordered.

Reply:
Probably not worth the computation, but we could use some checksum of the `data` value?

Reply:
This actually no longer exists since we're no longer exposing the higher level component!